<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MQTT Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditHost">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>test.mosquitto.org</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTopic">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>amr/topic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="editLog">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonConnect">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSubscribe">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Subscribe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPublish">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>publish</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPort">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="value">
     <number>1883</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>54</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>84</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>192</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
